tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
Looking online a bit it appears that there's an `isfinite(float)` function that could maybe be used instead.  I don't have access to the older compilers right now to check though.  Some of the information that I found implies that you're supposed to use `isfinite()` instead of comparing against `INFINITY` anyways.

Judging from online documentation, the closest in older VC++ implementations is going to be `_finite(double)`.  Looks like that provides the same functionality just under a different name.  It seems a bit ugly to wrap it in a macro just for Visual C, but I don't really know another solution and it's probably cleaner than excluding the entire file.  However, the functions in the file are unused unless FENV is enabled anyways, so how much does it actually matter if the file is compiled when FENV isn't active?